project/pyproject.toml.jinja

```diff
@@ -65,6 +65,9 @@ doc.shell = "cd docs && mkdocs serve -a localhost:8010"
 lint.help = "Run linters over all files"
 lint.cmd = "pre-commit run --all-files"
 
+# PDM hook to automatically sort dependencies when they change.
+post_install.shell = "pre-commit run keep-sorted --files pyproject.toml > /dev/null 2> /dev/null"
```

Comment:
Do the redirs to devnull mean that a failure like "disk full" or "keep-sorted plugin is not installed" will be hidden and `pdm install` will fail quite mysteriously? 

How bad is the keep-sorted stdout/stderr?

Reply:
I think in the case of `pdm install` on a full disk will continue to fail normally (which is to say, maybe quite badly? I, uh, am reluctant to test this specific case 😄) and we just wouldn't get to `post_install` hooks in those cases.

The `keep-sorted` stdout/stderr is confusing, especially since it comes with little context and users triggering `post_install` hooks are likely not thinking of running `keep-sorted`. I even saw some while working on https://github.com/quant-aq/aeromancy/pull/21/files#r1730341863 which were a bit jarring.